### PR TITLE
Improving the Custom Methods and Add Batch on get Method

### PR DESCRIPTION
**Pull request by davidcostadev**

### CODE REVIEW

File: src/service/Service.js [hunk 1]
```diff
@@ -60,18 +60,42 @@ const list = async ({ query }, Model, { options, database }) => {
...
+    batch,
+  } = selector({
+    batch: SelType.batchSelType,
+    ...filters,
+  }, query);
+
+  const select = {
+    where: { id }
+  };
+  if (batch) {
```
[the remark this answers is not in the document]
Comment (davidcostadev) [reply src/service/Service.js]: For this fix, i will create a issue for this

File: src/service/createService.js [hunk 2]
```diff
@@ -18,15 +19,19 @@ export const createResourceService = (model, {
...
-  only.forEach((action) => {
-    methods[action] = req => Service[action](req, model, { definitions, options, database });
-  });
 
-  return {
-    ...methods,
-    ...custom,
-  };
-};
+  const methods = only.reduce((pre, cur) => {
```
Comment (diasbruno) [inline src/service/createService.js]: `only` can be renamed to `actions`.

Comment (davidcostadev) [reply src/service/createService.js]: Only mean that I can choose which method only I want to generate in this service. In the future, I will implement that `except`, It'll mean that I want all methods except one for example

**Comment by davidcostadev** [comment]
I merged this PR on wrong branch, i will open another PR

**Pull request merged**